### PR TITLE
Adding button to launch fault counter to main screen

### PR DESCRIPTION
**Pull request by lisazacarias**
Currently the fault counter can only be launched from a direct command. Adding a button that launches it to the top right of the main cavity display next to the button that launches the decoder.

### CODE REVIEW

File: displays/cavity_display/cavity_display.py [hunk 1]
```diff
@@ -71,4 +72,14 @@ def __init__(self, parent=None, *args, **kwargs):
 
         self.groupbox = QGroupBox()
         self.groupbox.setLayout(self.groupbox_vlayout)
-        self.vlayout.addWidget(self.groupbox)
\ No newline at end of file
+        self.vlayout.addWidget(self.groupbox)
+
```
Comment (hmarts9) [inline displays/cavity_display/cavity_display.py:76]: Implementation of the fault counter button looks great overall. One thing I noticed is that this button follows the same pattern as the decoder button, would it be beneficial to maybe extract both into a helper method to reduce code duplication?

**Review by hmarts9 — APPROVED** [review]
I reviewed the changes and the implementation looks solid and maintains consistent styling. Overall, the code is really well structured and ready to merge. Nice work!

File: displays/cavity_display/frontend/fault_count_display.py [hunk 2]
```diff
@@ -41,57 +40,66 @@ def __init__(self):
...
 
-        end_text = QLabel("End:")
         self.end_selector = QDateTimeEdit()
         self.end_selector.setCalendarPopup(True)
 
         self.start_selector.setMinimumDateTime(min_date_time)
         self.start_selector.setDateTime(intermediate_time)
         self.end_selector.setDateTime(end_date_time)
+        self.start_selector.dateChanged.connect(self.update_plot)
+        self.start_selector.editingFinished.connect(self.update_plot)
```
Comment (Derikka) [inline displays/cavity_display/frontend/fault_count_display.py:60]: What's "editingFinished" do?

File: displays/cavity_display/cavity_display.py [hunk 3]
```diff
@@ -71,4 +72,14 @@ def __init__(self, parent=None, *args, **kwargs):
```
[the remark this answers is not in the document]
Comment (lisazacarias) [reply displays/cavity_display/cavity_display.py:65]: Good idea!

**Pull request merged**